tests/integration/spark/test_include_headers.py

```diff
@@ -41,6 +42,50 @@ def test_includes_function_with_filter_and_histo(self):
         # Compare the standard deviations of equivalent set of numbers
         self.assertEqual(histo.GetStdDev(), required_stdDev)
 
+        sc = SparkContext.getOrCreate()
+        sc.stop()
+
+    def test_extend_ROOT_include_path(self):
+        """
+        Check that the include path of ROOT is extended correctly to solve
+        headers that reference each other.
+        """
+        import ROOT
+
+        header_folder = (
+            "tests/integration/local/test_headers/"
+            "headers_folder"
+        )
+
+        PyRDF.use("spark")
+        PyRDF.include_headers(header_folder)
+
+        # Get list of include paths seen by ROOT
+        ROOT_include_path = ROOT.gInterpreter.GetIncludePath()
+        ROOT_include_path = ROOT_include_path.split(" ")
+
+        # Create new include folder token
+        new_folder_include = "-I\"{}\"".format(header_folder)
+
+        # Check that new folder is in ROOT include paths
```

Comment:
"Check that include directories of header files are in ROOT include paths", since it can be more that one directory.

Reply:
In this specific test is just one folder

Reply:
I thought [this](https://github.com/JavierCVilla/PyRDF/pull/45/files/6c9c4d1a427242c58573320e17f486d68b0ab6fe#diff-58891385664615c0581109a61cff5c8aR55) was a tuple, does it fit in one single line?

Reply:
yes, sorry, I was using a longer path before